### PR TITLE
Use a stored copy of freemarker.jar for the travis build

### PR DESCRIPTION
**Pull request by jdekonin**
* reduces travis failure due to external dependency
* #1341

Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -63,8 +63,7 @@ before_script:
   # - ccache -s -z
   # Exit immediately if any unexpected error occurs.
   - set -e
-  - wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
-  - tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
+  - wget https://ci.eclipse.org/openj9/userContent/freemarker.jar
```
Comment (fjeremic) [inline .travis.yml]: Any reason to remove the version information from the file name?

Comment (jdekonin) [reply .travis.yml]: Not really, and I was just thinking of putting that back in to make it obvious.  You're comment seals it.

**Comment by DanHeidinga** [comment]
@jdekonin Can you have this pull from the Eclipse CI and fallback to sourceforge if the CI version isn't available?

**Comment by jdekonin** [comment]
This commit verifies failover if freemarker.jar is not available from the eclipse userContent location; note it passes configure, the fail is in compile because my fork is apparently not up2date.
https://travis-ci.org/eclipse/openj9/builds/351489405
Note I have freemarker-2.3.9.jar which doesn't exist in userContent.
Next commit will revert the version number only.  Sorry for all the 'push' spam' to test.

**Comment by DanHeidinga** [comment]
Tested manually by using the wrong link for the first wget, the fallback works fine.

**Pull request merged**